apps/src/studio/studio.js

```diff
@@ -118,6 +118,9 @@ const EdgeClassNames = [
 let level;
 let skin;
 
+// These skins can be published as projects.
+const PROJECT_SKINS = ['gumball', 'studio'];
```

Comment:
Should this be `PUBLISHABLE_SKINS` instead? For example, I can go to studio.code.org/projects/infinity, so I would expect it to be in a list called PROJECT_SKINS